branding/pdf/entities.adoc

```diff
@@ -10,6 +10,7 @@
 :x86_64: x86_64
 :s390x: s390x
 :arm64: ARM64
+:almalinux: AlmaLinux
```

Comment:
Could probably make this just `alma`

Reply:
In this case I'd not do it. The name is "AlmaLinux" and not "Alma Linux". Besides I am not sure you want to have an entity which means "soul" in Spanish :-D

Reply:
lol, sure. Just trying to make the entities easier to handle and remember :)